### ISSUE TITLE
Feature request: Favorites

### ISSUE THREAD

**Issue by lfarkas** [wireapp/wire#98]
hi,
it'd be very useful if i can pin/favorites a few contact or group and those are always at the top of my chat lists.

**Comment by jprberlin** [comment]
Would this be global or device specific?

**Comment by lfarkas** [comment]
imho global, like favorites in your google contacts (they called starred)

**Comment by baimafeima** [comment]
Good idea!

**Comment by cardassian-tailor** [comment]
Yes. I think about this constantly when I go hunting for my 'chat with self' trick and its at the bottom of the list.
